apps/tasks/lib/components/patient_bottom_sheet.dart

```diff
@@ -0,0 +1,175 @@
+          builder: (BuildContext ctx) => Padding(
+            padding: const EdgeInsets.symmetric(horizontal: paddingMedium),
+            child: Column(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.symmetric(vertical: paddingMedium),
+                  child: Column(
+                    children: [
+                      Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                        children: [
+                          IconButton(
+                            icon: const Icon(Icons.close),
+                            onPressed: () {
+                              Navigator.of(context).pop();
+                            },
+                          ),
+                          Flexible(
+                            child: Padding(
+                              padding: const EdgeInsets.only(top: paddingOffset),
+                              child: Text(
+                                widget.task.patient.name,
+                                style: const TextStyle(
+                                  overflow: TextOverflow.ellipsis,
+                                  fontSize: fontSizeBig,
+                                ),
+                              ),
+                            ),
+                          ),
+                          IconButton(
+                            icon: const Icon(Icons.edit_outlined),
+                            onPressed: () {},
+                          ),
+                        ],
+                      ),
+                      SizedBox(
+                        width: width * 0.45,
+                        child: const Center(
+                          child: ExpansionTile(
+                            title: Text(
+                              overflow: TextOverflow.ellipsis,
+                              "Room 2 - Bed 1", // TODO: replace with real data
+                              style: TextStyle(color: Colors.grey),
+                            ),
+                            children: [
+                            ],
+                          ),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                TextFormField(
+                  initialValue: widget.task.notes,
+                  decoration: InputDecoration(
+                    hintText: context.localization!.notes,
+                  ),
+                  maxLines: 6,
+                  keyboardType: TextInputType.multiline,
+                ),
+                Padding(
+                  padding: const EdgeInsets.symmetric(vertical: paddingMedium, horizontal: paddingOffset ),
+                  child:  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      Text(context.localization!.tasks, style: const TextStyle(fontSize: fontSizeBig, fontWeight: FontWeight.bold),),
```

Comment:
Tasks should be capitalized here

Reply:
According to the mockups not

Reply:
![Frame 156 (1)](https://github.com/helpwave/mobile-app/assets/67233923/a88e6e13-7ae4-46bc-8ed0-f0df93dc811c)

Reply:
Sorry, I think was a misunderstanding